plasmapy/formulary/tests/test_parameters.py

```diff
@@ -27,7 +27,7 @@
     magnetic_energy_density,
     magnetic_pressure,
     upper_hybrid_frequency,
-    lower_hybrid_frequency,
+    lower_hybrid_frequency, Bohm_diffusion,
```

Comment:
For stylistic consistency...
```suggestion
    lower_hybrid_requency,
    Bohm_diffusion,
```
That way this addition shows up just as a new line in git.